lib/rust/parser/src/syntax/token/operator.rs

```diff
@@ -284,34 +281,13 @@ pub enum Precedence {
     Negation,
     Application,
     Annotation,
-    /// A value that is higher than the precedence of any operator.
-    Max,
+    // NOTE: The highest value must not exceed 0x7e--see usage of `into_u8`.
```

Comment:
I see only one usage of into_u8: in `ModifiedPrecedence::new`; there's no docs, and I don't know why 0x7f is not allowed.

Reply:
It's unlikely we would ever want to use high numeric values for these, but out of an abundance of caution I put a note here to make sure that anyone considering doing so knows what that could affect. If they are really determined to make that sort of change, the reference to `into_u8` points them in the right direction to find out why they can't use 0x7f.